src/Microsoft.ML.Data/Transforms/ValueMapping.cs

```diff
@@ -31,7 +31,39 @@
 
 namespace Microsoft.ML.Transforms
 {
-    /// <include file='doc.xml' path='doc/members/member[@name="ValueMappingEstimator"]/*' />
+
+    /// <summary>
+    /// Estimator for <see cref="ValueMappingTransformer"/>. It is a 1-1 mapping from one value to another value.
```

Comment:
Suggest: "Create a key-value map using the pairs of values in the input data" or something like that?